app/admin/RTMediaAdmin.php

```diff
@@ -179,33 +179,33 @@ public function install_transcoder_admin_notice() {
 			$show_notice = get_site_option( 'install_transcoder_admin_notice', 1 );
 			if ( '1' === $show_notice || 1 === $show_notice ) :
 				?>
-			<div class="notice notice-info install-transcoder-notice is-dismissible">
-				<?php wp_nonce_field( '_install_transcoder_hide_notice_', 'install_transcoder_hide_notice_nonce' ); ?>
-				<p>
-					<?php
-					$allowed_tags = array(
-						'a' => array(
-							'href'   => array(),
-							'target' => array(),
-						),
-					);
-					echo wp_kses( __( 'Install <a href="https://wordpress.org/plugins/transcoder/" target="_blank">Transcoder plugin</a> to convert audio/video files and thumbnails generation.', 'buddypress-media' ), $allowed_tags );
-					?>
-				</p>
-			</div>
-			<script type="text/javascript">
-				jQuery( document ).ready( function() {
-					jQuery( '.install-transcoder-notice.is-dismissible' ).on( 'click', '.notice-dismiss', function() {
-						var data = {
-							action: 'install_transcoder_hide_admin_notice',
-							install_transcoder_notice_nonce: jQuery('#install_transcoder_hide_notice_nonce').val()
-						};
-						jQuery.post( ajaxurl, data, function ( response ) {
-							jQuery('.install-transcoder-notice').remove();
+				<div class="notice notice-info install-transcoder-notice is-dismissible">
+					<?php wp_nonce_field( '_install_transcoder_hide_notice_', 'install_transcoder_hide_notice_nonce' ); ?>
+					<p>
+						<?php
+						$allowed_tags = array(
+							'a' => array(
+								'href'   => array(),
+								'target' => array(),
+							),
+						);
+						echo wp_kses( __( 'Install <a href="https://wordpress.org/plugins/transcoder/" target="_blank">Transcoder plugin</a> to convert audio/video files and thumbnails generation.', 'buddypress-media' ), $allowed_tags );
+						?>
+					</p>
+				</div>
+				<script type="text/javascript">
```

Comment:
We can move these JS codes in to a separate/existing JS file to make the php code clean.

Reply:
as of now we are not changing code structure for this release